### ISSUE TITLE
[Bug] Rounded Corner hull block cannot stretch further than 12m in the Y direction

### ISSUE THREAD

**Issue by ProPeach** [tsunamayo/Starship-EVO#6040]
### Short Description of the issue:
No matter what scale you start at, this block cannot stretch further than 12m tall
![image](https://github.com/user-attachments/assets/ad794cc9-a251-4a06-be9e-29d2d662a629)
![image](https://github.com/user-attachments/assets/af56f6a0-52dc-4e0a-9afe-284856b1b97b)

### Steps to reproduce the issue:
- Grab a Rounded Corner block
- Stretch it out at different scales

### Environment
- Starship EVO version: 24w41a
- Graphics Card: RX6700XT

**Comment by TIKIRobo** [comment]
A lot of the rounded shapes have weird hitbox issues(#5456 #5322 #5230), maybe this is related?

**Comment by tsunamayo** [comment]
Yeah those shape are a bit borked collisions wise, it is a bit time consuming to fix unfortunately. It is really hard to make collision work with stretching and those rounded (sometime non-convex, although this particular shape is convex).
